README.md

```diff
@@ -1,8 +1,10 @@
 # Codacy meta-repo
```

Comment:
As a reference this is great. I was thinking that it would be cool if we had a "what if I wanted to add a GitHub workflow?", or a "What if I wanted to add a Code owners file?". What I mean, is a step by step guide of how to use this to perform some common operation to all our repos

Reply:
I like it, can I add that on a separate future PR and focus this on documenting what it does at the moment?

Reply:
Sure 👍